src/actions/index.js

```diff
@@ -217,12 +217,25 @@ export const trashFiles = files => {
         trashed.push(await cozy.client.files.trashById(file.id))
       }
     } catch (err) {
-      return dispatch({
-        type: TRASH_FILES_FAILURE,
-        alert: {
-          message: 'alert.try_again'
+      const isAlreadyInTrash = err => {
```

Comment:
Is there any specific reason why the function is declared inside another function? If not I feel like it should be declared outside, to avoid recreating it every time.